src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,78 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private final Map<String, List<Integer>> transactions = new HashMap<>();
+    private final Map<String, String> results = new HashMap<>();
+
+    public void getStatistic(String inputFileName, String outputFileName) {
```

Comment:
This is how `getStatistic()` should look like:

![image](https://user-images.githubusercontent.com/20326858/230874758-cc49e319-6ab7-4777-8dc7-5b7eebfca13e.png)